### ISSUE TITLE
Crash in `rules/linebreaks`

### ISSUE THREAD

**Issue by coderaiser** [stylelint-stylistic/stylelint-stylistic#56]
### What minimal example or steps are needed to reproduce the bug?

```js
file:///Users/coderaiser/cloudcmd/node_modules/@stylistic/stylelint-plugin/lib/rules/linebreaks/index.js:2
import { rule as _rule, Input } from "postcss"
                        ^^^^^
SyntaxError: Named export 'Input' not found. The requested module 'postcss' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'postcss';
const { rule: _rule, Input } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:180:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:263:5)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:578:26)

Node.js v22.14.0
```

### What minimal configuration is needed to reproduce the bug?

default config

### How did you run Stylelint?

api

### Which Stylelint-related dependencies are you using?

latest

### What did you expect to happen?

No crash

### What actually happened?

Crash

### Do you have a proposal to fix the bug?

Use import default, as suggested by node.js

**Comment by MorevM** [comment]
Good day @coderaiser 

Can you provide a reproduction?
I've just tested locally and within an online container and I see no unexpected errors.

You can start from here: https://stackblitz.com/edit/stackblitz-starters-qddfq4mi
Run `npm run lint` to see that everything works as expected.

**Comment by coderaiser** [comment]
Looks like I have old version of postcss (v7), when I add postcss v8 as direct devDependency it works good. Thank you!

**Comment by MorevM** [comment]
That's interesting, in that case we need to add `postcss` with the correct version as `peerDependency` to the package.
Or, if that's the only problem with `postcss@7` and your PR fixes it, it might be worth doing.

I'm a bit limited in time right now - I'll leave the issue open so I don't forget, I'll have a look at it later and decide what to do :)

Thanks for noticing!

**Comment by coderaiser** [comment]
Yes, both solutions would be great, but `peerDependency` is more future proof, since in my case `postcss` was dependency of `css-loader@3.6.0`, so `stylelint-stylistic` can install it's own version of `postcss`, or at least submit warning :).
